app/views/drivers/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Driver Details</h2>
+
+<nav>
+  <%=link_to "Homepage", homepages_path %> |
+  <%= link_to "Return to Driver List", drivers_path %> |
+<!--  Need to be able to change availability-->
+
+</nav>
+<p>Driver Name: <%= @drivers.name %></p>
+<p>VIN: <%= @drivers.vin %></p>
+<p>Driver ID: <%= @drivers.id %></p>
+<p>Available: <%= @drivers.available %></p>
+<p>Driver Avg. Rating: <%= @drivers.mean_rating %></p>
+<p> Driver Total Earnings: <%= cents_dollars(@drivers.total_earnings) %></p>
+
+<ul>
+  <% @drivers.trips.each do |trip| %>
+    <li><strong><%= link_to trip.passenger.name, passenger_path(trip.passenger.id) %></strong> : Trip ID: <%=trip.id %> | Trip Rating : <%=trip.rating %> | Passenger ID: <%=trip.passenger.id%> | Cost: <%=cents_dollars(trip.cost) %> | <%= link_to "Delete Trip", trip_path(trip.id), method: :delete, data: {confirm: "Are you sure?"} %> |</li>
+  <% end %>
+</ul>
```

Comment:
Notice that you do this a **lot**.  You can make this a [partial](https://stackoverflow.com/questions/33637972/how-to-render-partial-view) and use it on all the show pages or anyplace you need a list of trips.  

You can also check the [documentation of the rails api](https://api.rubyonrails.org/classes/ActionView/PartialRenderer.html)